src/main/java/com/dungeoncrawler/Javiarenka/equipment/EquipmentSlots.java

```diff
@@ -10,6 +10,8 @@
     ARMS("Arms"),
     BAGGAGE("Baggage");
 
+    //todo head and legs
+
```

Comment:
Mamy gdzieś na to ticketa?

Reply:
Utworzyłam ticketa na stworzenie slota na head, legs i rings.